doc/nrf/protocols/thread/tools.rst

```diff
@@ -87,7 +87,7 @@ To program the nRF device with the RCP application, complete the following steps
 
          .. code-block:: console
 
-            west build -p always -b nrf52840dongle_nrf52840 nrf/samples/openthread/coprocessor/ -- -DOVERLAY_CONFIG="overlay-usb.conf" -DDTC_OVERLAY_FILE="usb.overlay"
+            west build -p always -b nrf52840dongle_nrf52840 nrf/samples/openthread/coprocessor/
```

Comment:
Isn't `-S usb` missing here?

Reply:
this place has been omitted in this PR: https://github.com/nrfconnect/sdk-nrf/pull/11593
Extra USB configuration is not needed anymore for dongle